save_folder: '/media/armando/LaCie/robot_data'
save_folder_default: '/media/armando/DATA/robot_data'